server.py

```diff
@@ -1,42 +1,57 @@
-import socket
+import socket as s
 
-ADDR = ('127.0.0.1', 8000)
-socket = socket.socket(
-    socket.AF_INET, socket.SOCK_STREAM, socket.IPPROTO_IP
+
+def response_ok(reason_phrase):
+    return "HTTP/1.1 200" + reason_phrase + "\r\n"
+    "Content-Type: text/html; charset=utf-8\r\n"
+
+
+def response_error(error_code, reason_phrase):
+    response = "HTTP/1.1 " + error_code + " " + reason_phrase + "\r\n"
```

Comment:
I'd rather see you use string formatting over concatenation.  It's better to do so because of all the little intermediary strings that get created with concatenation.